apps/explorer/vite.config.ts

```diff
@@ -5,16 +5,33 @@ import { pathAlias } from '@mysten/core/vite.config';
 import react from '@vitejs/plugin-react';
 import { defineConfig } from 'vite';
 import svgr from 'vite-plugin-svgr';
+import viteSentry from 'vite-plugin-sentry';
 
 // Assign the Vercel Analytics ID into a vite-safe name:
 process.env.VITE_VERCEL_ANALYTICS_ID = process.env.VERCEL_ANALYTICS_ID;
 
+const authToken = process.env.SENTRY_AUTH_TOKEN;
+const dryRun = process.env.NODE_ENV !== 'production' || !authToken;
+
 // https://vitejs.dev/config/
 export default defineConfig({
-    plugins: [react(), svgr()],
+    plugins: [
+        react(),
+        svgr(),
+        viteSentry({
```

Comment:
I'm curious if we need to do this at all, I imagine if we just build and expose sourcemaps that the Sentry client should be able to automatically fetch it instead of requiring us to upload it? I'm not 100% sure but it could be worth checking (although it would be a little hard to verify right now because of the Sentry cap in place right now)

Reply:
Makes sense let me check